microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -518,6 +518,24 @@ def verify_yum_conf(self, node: Node) -> None:
         else:
             raise SkippedException(f"Unsupported distro type : {type(node.os)}")
 
+    @TestCaseMetadata(
+        description="""
+        Verify if there is any issues in and after 'yum update'
+
+        Steps:
+        1. Run 'yum update -y' command.
+        2. Reboot the VM and see if the VM is still in good state.
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+    )
+    def verify_yum_update(self, node: Node) -> None:
+        if isinstance(node.os, Fedora):
```

Comment:
don't restrict the os, mayby just check whether it is Posix

Reply:
Replaced "yum update -y" with node.os.update_packages("")

Reply:
@SRIKKANTH please fix CI check error

Reply:
@LiliDeng error is fixed now.